gtimelog.appdata.xml

```diff
@@ -1,29 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<application>
-    <id type="desktop">gtimelog.desktop</id>
+<component type="desktop-application">
+    <id>org.gtimelog.GTimeLog</id>
```

Comment:
I noticed that `appstreamcli validate --strict --explain --pedantic gtimelog.appdata.xml` says

```
P: org.gtimelog.GTimeLog:3: cid-contains-uppercase-letter org.gtimelog.GTimeLog
   The component ID should only contain lowercase characters.
```

Reply:
yeah there is a lot of stuff it will say that are IMHO irrelevant. This is one of them. The spec written by the same people say to not change the ID.

For a different reason I would say to NOT change the ID for that reason. And I think the app id pass to GApplication should also be synced with this.

Reply:
Also renaming the .desktop file and icons to match and have all of these installed by the makefile.